src/Codegen/Constraints/StringBuilder.php

```diff
@@ -87,27 +85,17 @@ protected function getCheckMethod(): CodegenMethod {
     $sanitize = $this->typed_schema['sanitize'] ?? null;
     $sanitize_string = $this->ctx->getSanitizeStringConfig();
     if ($sanitize is nonnull && $sanitize_string === null) {
-      throw new \Exception(
-        "Specified `sanitize` on a string without providing sanitization functions.",
-      );
+      throw new \Exception("Specified `sanitize` on a string without providing sanitization functions.");
     } else if ($sanitize is nonnull && $sanitize_string is nonnull) {
       if ($sanitize['multiline']) {
-        $sanitization_func = (string)$sanitize_string['multiline'];
+        $sanitization_func = get_function_name_from_function($sanitize_string['multiline']);
       } else {
-        $sanitization_func = (string)$sanitize_string['uniline'];
+        $sanitization_func = get_function_name_from_function($sanitize_string['uniline']);
       }
 
       $hb
-        ->addAssignment(
```

Comment:
hackfmt had some strong opinions here (and in some other places) since the line length of this expression is less than 120 chars. I hope that this is not an issue. I tried to maintain formatting (except for hhast autofixes) where possible. Sadly, a couple of them slipped in.